python-api/src/main/python/tests/client_test.py

```diff
@@ -38,39 +36,38 @@
 def mock_and_validate_create_new_session(defaults):
     global client_test
     create_session_request_mock_uri = base_uri + "/sessions/"
-    app_name = 'Test Client App'
+    app_name = 'Test App'
     conf_dict = {'spark.app.name': app_name}
-    responses.add(responses.POST, create_session_request_mock_uri,
-                  json={u'kind': u'pyspark', u'log': [], u'proxyUser': None,
-                        u'state': u'starting', u'owner': None, u'id':
-                            session_id}, status=201,
-                  content_type='application/json')
-    client_test = HttpClient(urlparse(base_uri), conf_dict=conf_dict,
-                             load_defaults=defaults)
-    assert isinstance(client_test, HttpClient)
-    assert client_test._config.get(client_test._CONFIG_SECTION,
-                                   'spark.app.name') == app_name
+    responses.add(
+        responses.POST, create_session_request_mock_uri,
+        json={u'kind': u'pyspark', u'log': [], u'proxyUser': None,
+              u'state': u'starting', u'owner': None, u'id': session_id},
```

Comment:
again you're using alignment.

Also, for a complicated parameter like this `json` one, I'd declare it outside the call and reference the variable.